internal/services/network/migration/network_packet_capture.go

```diff
@@ -38,6 +39,13 @@ func (NetworkPacketCaptureV0ToV1) Schema() map[string]*pluginsdk.Schema {
 			ForceNew: true,
 		},
 
+		"target_type": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ForceNew: true,
+			Default:  string(network.PacketCaptureTargetTypeAzureVM),
+		},
+
```

Comment:
All the changes within this file need to be reverted, state migrations are point in time references, so this will break things

Reply:
Updated